Uppsala/UppEt10.xml

```diff
@@ -0,0 +1,445 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="UppEt10">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title xml:id="t1">
+                    ʾArgānona Wəddāse
+                </title>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0.
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0356Upp"/>
+                        <collection>O Etiop.</collection>
+                        <idno>O Etiop. 10</idno>
+                        <altIdentifier><idno>Löfgren 10</idno></altIdentifier>
+                        <altIdentifier><idno>Zetterstéen X</idno></altIdentifier>
+                        <altIdentifier><idno>Äthiop. Ms. 5</idno></altIdentifier>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1ra" to="151rb"/>
+                            <title type="complete" ref="LIT1146Argano"/>
+                            <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">43–44</citedRange></bibl>
+                                provides a general introduction to the work, discussing the number of manuscripts available in Europe.
+                            </note>
```

Comment:
This information is not necessary in a manuscript record. If the information is valuable, it should better be given in the work record of LIT1146Argano, either explicitely or as a bibl note.